main.js

```diff
@@ -0,0 +1 @@
+alert("hello");
```

Comment:
In javascript, use single quotes instead of double quotes as best practice.

Reply:
Change Complete